File: Java/bee 1789.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        while(sc.hasNext()) {
            int n = sc.nextInt();

            int[] velocidades = new int[n];
            for(int i = 0; i < n; i++) {
                velocidades[i] = sc.nextInt();
            }

            System.out.println(classificarNivel(getMaior(velocidades)));

        }
    }

    private static int getMaior(int[] arr) {
        int maior = arr[0];

        for(int i = 0; i < arr.length; i++) {
            if(arr[i] > maior) {
                maior = arr[i];
            }
        }

        return maior;
    }

    private static int classificarNivel(int n) {
        if(n < 10) {
            return 1;
        } else if(n < 20) {
            return 2;
        } else {
            return 3;
        }
    }
}

File: C++/bee 1789.cpp
#include <iostream>
using namespace std;

int getMaior(int arr[], int arrayLen) {
    int maior = arr[0];
    for(int i = 0; i < arrayLen; i++) {
        if(arr[i] > maior) {
            maior = arr[i];
        }
    }

    return maior;
}

int main() {
    int n;
    while(cin >> n) {
        int velocidades[n];
        for(int i = 0; i < n; i++) {
            int v;
            cin >> v;
            velocidades[i] = v;
        }

        int maior = getMaior(velocidades, n);

        if(maior < 10) {
            cout << 1 << endl;
        } else if(maior < 20) {
            cout << 2 << endl;
        } else {
            cout << 3 << endl;
        }
    }
    return 0;
}
